api/python/cellxgene_census/src/cellxgene_census/_open.py

```diff
@@ -334,4 +335,8 @@ def download_source_h5ad(dataset_id: str, to_path: str, *, census_version: str =
         anon=True,
         cache_regions=True,
     )
-    fs.get_file(locator["uri"], to_path)
+    fs.get_file(
+        locator["uri"],
+        to_path,
+        callback=TqdmCallback(),
```

Comment:
Since `TqdmCallback()` invocation is a side effect would suffice to mock `TqdmCallback()` in the test and simply assert that it is called?